airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -347,11 +348,16 @@ def create_stream_job(self, query: str, url: str) -> Optional[str]:
                         f"The stream '{self.name}' is not queryable, "
                         f"sobject options: {self.sobject_options}, error message: '{error_message}'"
                     )
+                elif (
```

Comment:
this just swallows the error and makes the sync succeed. it should actually raise a config error .